### ISSUE TITLE
wpt run for Chrome throws Python Attribute Error exception

### ISSUE THREAD

**Issue by musgravejw** [w3c/web-platform-tests#7854]
Hi there, trying to get the test suite to run on macOS Sierra v10.12.6 with Python 2.7.11.

Running `./wpt run chrome` on something like the example `dom/historical.html` throws the exception "Python quit unexpectedly", with the following output to the console:

```
 0:14.88 LOG: Thread-TestrunnerManager-1 INFO Starting runner
 0:44.37 LOG: Thread-Log INFO STDERR: Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 1073, in run
    self.function(*self.args, **self.kwargs)
  File "/Users/musgravejw/Development/oss/w3c/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 209, in init_timeout
    self.send_message("init_failed")
  File "/Users/musgravejw/Development/oss/w3c/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 203, in send_message
    self.command_queue.put((command, args))
AttributeError: 'BrowserManager' object has no attribute 'command_queue'
```

with `--log-mach-level debug`:

```
Using webdriver binary /Users/musgravejw/Development/w3c/web-platform-tests/_venv/bin/chromedriver
 0:01.14 LOG: MainThread INFO Updating test manifest /Users/musgravejw/Development/w3c/web-platform-tests/MANIFEST.json
 0:19.22 LOG: MainThread INFO Using 1 client processes
 0:19.29 LOG: MainThread INFO Starting http server on web-platform.test:8000
 0:19.29 SUITE_START: MainThread 1
 0:19.29 LOG: MainThread INFO Running reftest tests
 0:19.31 LOG: MainThread INFO Starting http server on web-platform.test:8001
 0:19.31 LOG: MainThread INFO Starting http server on web-platform.test:8443
 0:19.36 LOG: MainThread DEBUG Using 1 processes
 0:19.37 LOG: MainThread INFO No reftest tests to run
 0:19.37 LOG: MainThread DEBUG Stop flag set in ManagerGroup
 0:19.37 LOG: MainThread INFO Running wdspec tests
 0:19.37 LOG: MainThread DEBUG Using 1 processes
 0:19.37 LOG: MainThread INFO No wdspec tests to run
 0:19.37 LOG: MainThread DEBUG Stop flag set in ManagerGroup
 0:19.37 LOG: MainThread INFO Running testharness tests
 0:19.37 LOG: MainThread DEBUG Using 1 processes
 0:19.38 LOG: Thread-TestrunnerManager-1 DEBUG Dispatch start_init
 0:19.38 LOG: Thread-TestrunnerManager-1 DEBUG new state: initializing_browser
 0:19.38 LOG: Thread-TestrunnerManager-1 DEBUG Dispatch init
 0:19.38 LOG: Thread-TestrunnerManager-1 DEBUG Init called, starting browser and runner
 0:19.38 LOG: Thread-TestrunnerManager-1 DEBUG Starting browser with settings {}
 0:19.39 LOG: Thread-TestrunnerManager-1 DEBUG Waiting for server to become accessible: http://127.0.0.1:4451/
 0:19.43 PROCESS_OUTPUT: ProcessReader (pid:42953) Full command: /Users/musgravejw/Development/w3c/web-platform-tests/_venv/bin/chromedriver --port=4451 --url-base=/
(pid:42953) "Starting ChromeDriver 2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2) on port 4451"
 0:19.43 PROCESS_OUTPUT: ProcessReader (pid:42953) "Only local connections are allowed."
 0:19.89 LOG: Thread-TestrunnerManager-1 INFO Starting runner
 0:19.90 LOG: Thread-TestrunnerManager-1 DEBUG Test runner started
 0:19.91 LOG: Thread-TestrunnerManager-1 DEBUG Executor setup
```

Issue referenced here:  https://github.com/w3c/web-platform-tests/issues/6998

**Comment by annevk** [comment]
@jgraham who can help with this?

@musgravejw can you run tests yourself with `./wpt serve`?

**Comment by annevk** [comment]
(This blocks https://github.com/whatwg/html/pull/3084.)

**Comment by jeffcarp** [comment]
I believe #5774 would fix this.

**Comment by jgraham** [comment]
Merged #5774, but if you keep getting this something else is wrong since it only happens when the browser failed to start.
